src/__tests__user__/insert.test.ts

```diff
@@ -0,0 +1,47 @@
+import mongoose from "mongoose";
+import User, { IUser } from "../models/User";
+
+describe("User model", () => {
+  beforeAll(async () => {
+    await mongoose.connect("mongodb://localhost:27017", {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+      useCreateIndex: true,
+    });
+  });
+
+  afterAll(async () => {
+    await User.findOneAndDelete({ token: "Test token" });
+    await mongoose.connection.close();
+  });
+
+  it("If the githubId is not entered, I get an error", async () => {
+    try {
+    } catch (error) {
+      expect(error).toThrow(`Path githubId is required.`);
+    }
+  });
+
+  it("Should save a user", async () => {
+    const user: IUser = new User({
+      githubId: "Test github Id",
+      username: "Test username",
+      token: "Test token",
+    });
+    const spy = jest.spyOn(user, "save");
+    const isgithubId = user.githubId;
+    const isUsername = user.username;
+    const istoken = user.token;
```

Comment:
What is the point of these variables?

Reply:
I'll check if the value is what it expects. I used Object.is to check for strict equality.